examples/README.md

```diff
@@ -51,6 +51,7 @@ relevant code.
 
 ### MCMC with gradients
 - [Hamiltonian MCMC](./sampling-hamiltonian-mcmc.ipynb)
+- [No-U-Turn MCMC](./sampling-nuts-mcmc.ipynb)
```

Comment:
Very minor but could we keep this alphabetical.

Reply:
done